fhir-server-test/src/test/resources/testdata/import-operation/test-import-skip.ndjson

```diff
@@ -0,0 +1,9 @@
+{"resourceType":"Patient","id":"72b0d93c-93d0-43d9-94a8-d5154ce07152","meta":{"versionId":"1","lastUpdated":"2020-04-17T00:09:36.910431Z"},"name":[{"family":"Doe","given":["import-1"]}],"telecom":[{"system":"phone","value":"555-1122","use":"home"}],"birthDate":"1970-01-01"}
```

Comment:
whats the provenance of this sample data.  just a rehash of what we already have out there?

Reply:
exactly, Dag had one that did an update. I originally created the sample data, and wanted a slight difference on this one